exercise-src/react-native/16-security/01-problem/src/services/auth/AuthProvider.tsx

```diff
@@ -14,10 +14,14 @@ GoogleSignin.configure({
 })
 
 const AuthProvider: FC<{ children: ReactNode }> = ({ children }) => {
-  const [userInfo, setUserInfo] = useState<UserInfo | null>()
+  const [userInfo, setUserInfo] = useState<UserInfo | undefined>()
 
   // Exercise: Implement `signIn` and `signOut` using `useCallback`.
 
+  const signIn = async () => {}
```

Comment:
Perfect. For the problem, do _not_ add the `useMemo` like the linter is suggesting. They will add this in the solution.